pokemontcgsdk/card.py

```diff
@@ -1,45 +1,45 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-# This file is part of pokemontcgsdk.
-# https://github.com/PokemonTCG/pokemon-tcg-sdk-python
-
-# Licensed under the MIT license:
-# http://www.opensource.org/licenses/MIT-license
-# Copyright (c) 2016, Andrew Backes <backes.andrew@gmail.com>
-
-import json
```

Comment:
This attribution seemed unnecessary and hard to copy to more classes that I was writing... I left a copy in `pokemontcgsdk/__init__.py` and removed it everywhere else. I'm open to reverting this... but didn't know how to handle the Copyright mentioning an individuals name specifically...